docs/zkapps/tutorials/11-advanced-account-updates.mdx

```diff
@@ -11,48 +20,42 @@ zkApps can now be deployed to Berkeley Testnet.
+  If the zkApp Account is set up with the `proof` permission, the account update proof must pass the verification key in the account.
+2. The account update must set `mayUseToken` to a truthy value (a value that evaluates to true in a Boolean context). 
 
-1. The account update must meet the criteria of the permissions on the zkApp account itself. If the zkApp Account is set up with the `proof` permission, the Account Update proof must pass the verification key in the account.
-2. The account update must set `mayUseToken` to a truthy value. However, to be allowed to do so, a transaction must get its account update “approved” by the corresponding manager account. The manager account can approve accounts based on its `access` permission.
+  However, for this to happen, a transaction must get its account update approved by the corresponding zkApp manager account. The zkApp manager account can approve accounts based on its `access` permission.
 
 ## zkApp Token Management
 
-As an example, you can write a token that can approve a zkApp account to send tokens from itself to another zkApp.
+For example, you can write a token that can approve a zkApp account to send tokens from itself to another zkApp.
 
-To implement this, you would have 3 contracts:
+To implement, you need three contracts:
 
-1. `MyToken`: The token contract. On instantiation, serves as the zkAppManagerContract for all instances of the token.
-2. `TokenUser`: A zkApp contract that manages tokens. This one is instantiated on the Mina (default) tokenId.
-3. `TokenHolder`: A zkApp contract that stores tokens on behalf of the `TokenUser` contract. This one is instantiated on the `MyToken` instance TokenId.
+1. `MyToken`: The token contract. On instantiation, it serves as the zkAppManagerContract for all instances of the token.
+2. `TokenUser`: A zkApp contract that manages tokens. This contract is instantiated on the Mina (default) `tokenId`.
+3. `TokenHolder`: A zkApp contract that stores tokens on behalf of the `TokenUser` contract. This contract is instantiated on the `MyToken` instance tokenId.
 
-The full code is provided in the [examples/zkapps/11-advanced-account-updates/](https://github.com/o1-labs/docs2/tree/main/examples/zkapps/11-advanced-account-updates/) file.
+The full code is provided in the [examples/zkapps/11-advanced-account-updates/](https://github.com/o1-labs/docs2/tree/main/examples/zkapps/11-advanced-account-updates/) folder.
 
-As shown in the example file, a token standard library abstracts many of these details away for standard use. The illustration here helps explain what that library is doing and how to think about zkAppManagerAccounts in detail.
+As shown in the example file, a token standard library abstracts many of these details away for standard use. This tutorial helps you understand what the library is doing and how to think about zkApp manager account.
```

Comment:
@Trivo25  which "example file" are we referring to here? can you help?

Reply:
As per my comment above @barriebyron